cities/crud.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from cities import models, schemas
+
+
+async def create_city(
+    db: AsyncSession,
+    city: schemas.CityCreate
```

Comment:
Ensure that `schemas.CityCreate` is correctly defined and imported. This schema should match the expected input data for creating a `City` instance.